proof/invariant-abstract/RISCV64/ArchKHeap_AI.thy

```diff
@@ -761,6 +761,30 @@ lemma state_hyp_refs_of_tcb_state_update:
   apply (clarsimp simp add: state_hyp_refs_of_def obj_at_def split: option.splits)
   done
 
+lemma state_hyp_refs_of_tcb_domain_update:
+  "kheap s t = Some (TCB tcb) \<Longrightarrow>
+   state_hyp_refs_of (s\<lparr>kheap := (kheap s)(t \<mapsto> TCB (tcb\<lparr>tcb_domain := d\<rparr>))\<rparr>)
+     = state_hyp_refs_of s"
```

Comment:
where is this style coming from? the point of right-hand operator wrapping is so we don't need to add extra indentation, otherwise we'd just stick it on the previous line; for this and the next two lemmas the `=` should go under `state`

Reply:
It looks like I was copying the existing lemmas that are just before these ones. I agree that this is obviously the wrong indentation but I don't want to fix those other lemmas in all architectures in this PR, so do we prefer correct indentation or local consistency?

Reply:
Maybe note the files down somewhere and then handle in another PR then.

Reply:
I ended up just fixing up the style for both my new lemmas and the ones I was copying. There's plenty of other horribly styled lemmas around here but I'll leave fixing them for next time they're touched.